test("Missing visit url. Error thrown", ()=>{

    try{
        
        // try code
        
    } catch(error){
        
        // Error code
    }
    
});

test("Present url", ()=>{
    
    // try code
    
});